backend/src/shu/api/auth.py

```
@@ -942,18 +539,29 @@ async def create_user(
+                auth_method=request.auth_method,
                 role=request.role,
                 is_active=True  # Admin-created users are active
             )
             db.add(user)
-            await db.commit()
-            await db.refresh(user)
+            
+            try:
+                await db.commit()
+                await db.refresh(user)
+            except IntegrityError as e:
```

Ruff (0.14.14)

Undefined name `IntegrityError`

(F821)